create-quasar/templates/app/quasar-v2/js-webpack/BASE/_.npmrc

```diff
@@ -0,0 +1,2 @@
+shamefully-hoist=true
```

Comment:
I'd say `.npmrc` file should be scaffolded only when using PNPM
But I guess the problem here is that Quasar asks about the package manager to use only after the scaffolding process took place

What about adding at least a comment on the top to mark this is only useful when using PNPM and can be deleted otherwise?

Reply:
I also thought about that. Then, I thought some people may want to migrate to PNPM, or it could be part of a project where multiple package managers are welcome(_open-source apps, others?_). Also, as you said the package manager is asked after scaffolding. Then, I decided to always include it since it's not a big deal.

We can put `utils.runningPackageManager` to `scope`, then use it when it's defined. Then, we can scaffold this file when it's PNPM or undefined. But, I guess adding comments would be a simpler addition, and would still cover all of the things I mentioned. So, I will add a comment.